### PR TITLE
Update devnet.mdx

### PR DESCRIPTION
**Pull request by Kay-Zee**
Closes: N/A

## Description

Change to using unified access address: access.devnet.nodes.onflow.org

### CODE REVIEW

**Review by turbolent — APPROVED** [review]
Nice!

**Comment by turbolent** [comment]
@Kay-Zee Is this also ready for mainnet? (https://github.com/onflow/flow/blob/master/docs/content/concepts/flow-concepts/mainnet.mdx)

**Comment by Kay-Zee** [comment]
not yet, will be after next mainnet spork

**Review by vishalchangrani — APPROVED** [review]

File: docs/content/concepts/flow-concepts/devnet.mdx [hunk 1]
```diff
@@ -5,15 +5,15 @@ title: Flow Devnet
 
 Currently, there is an internal Devnet, available for access:
 
-- [access-001.devnet16.nodes.onflow.org:9000](http://access-001.devnet16.nodes.onflow.org:9000/)
+- [access.devnet.nodes.onflow.org:9000](http://access.devnet.nodes.onflow.org:9000/)
```
Comment (psiemens) [inline docs/content/concepts/flow-concepts/devnet.mdx:8]: This is great news!

**Review by psiemens — APPROVED** [review]

**Pull request merged**